lib/elements/PixelifyHooks.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useRef } from 'react';
+import paintPixels from '../helpers/paintPixels';
+
+const PixelifyHook = ({ src, width, height, pixelSize, centered, fillTransparencyColor }) => {
+	const canvasRef = useRef(null);
+
+	useEffect(() => {
+		const parsedPixelSize = parseInt(pixelSize, 10);
+		console.log(parsedPixelSize);
```

Comment:
hey, can you remove the console log? Thanks!

Reply:
Guess I missed one 👍